lib/presentation/mixins/go_to_direct_chat_mixin.dart

```diff
@@ -43,11 +44,12 @@ mixin GoToDraftChatMixin {
         contactPresentationSearch: contactPresentationSearch,
       );
     } else {
-      TwakeDialog.showFutureLoadingDialogFullScreen(
+      ActiveRoomIdNotifier().value = roomId;
```

Comment:
context.push here is not enough ?, okay i can pair with you in the afternoon

Reply:
oh okay, i understand your concern, you want to keep history and also change of `activeChatId` in chat list